conf/application.conf

```diff
@@ -28,8 +28,6 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoModule"
 play.modules.enabled += "config.Module"
 
-play.http.filters = "uk.gov.hmrc.play.bootstrap.backend.filters.BackendFilters"
-
 play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandler"
```

Comment:
@CarolineS-QA Can we add in the filter for session id as per the bootstrap-play guide:
`play.filters.enabled += "uk.gov.hmrc.play.bootstrap.frontend.filters.SessionIdFilter"`

Reply:
@adamconder @CarolineS-QA it's a backend so I don't think this is needed - I think it comes in backend.conf

Reply:
I thought since it was a backend filter we didn't need to change anything? Although if the default is false and you want it true, then okay.

Reply:
Interesting! Thanks for confirming Brian.